subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformIntegrationTest.groovy

```diff
@@ -2324,6 +2324,8 @@ Found the following transforms:
             ${declareTransform(transformImplementation)}
 
             task resolve(type: Copy) {
+                duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
The `Copy` task should not be affected by the change.

Reply:
Done